iocore/net/quic/QUICAckFrameCreator.h

```diff
@@ -69,7 +69,7 @@ class QUICAckFrameManager : public QUICFrameGenerator
     bool _available                         = false; // packet_number has data to sent
     bool _should_send                       = false; // ack frame should be sent immediately
     bool _has_new_data                      = false; // new data after last sent
-    uint32_t _ack_eliciting_count           = 0;     // every two ack-eliciting packet should send ack immediatly
+    uint32_t _ack_eliciting_count           = 0;     // every two ack-eliciting packet should send ack imeediately
```

Comment:
imeediately

Reply:
that's embarrassing! :) Thanks for the catch!